config/packages/hwi_oauth.yaml

```diff
@@ -1,70 +1,11 @@
 hwi_oauth:
-    # list of names of the firewalls in which this bundle is active, this setting MUST be set
-    firewall_names: [main]
-#    fosub:
-#        properties:
-#            facebook: providerId
-#            google: providerId
-#            auth0: providerId
-#            linkedin: providerId
-#            github: providerId
-#            gitlab: providerId
-#            twitter: providerId
-#            instagram: providerId
-
-    # https://github.com/hwi/HWIOAuthBundle/blob/master/Resources/doc/2-configuring_resource_owners.md
+    # https://github.com/hwi/HWIOAuthBundle/blob/master/docs/2-configuring_resource_owners.md
     resource_owners:
         facebook:
             type:                facebook
             client_id:           '%env(FB_ID)%'
             client_secret:       '%env(FB_SECRET)%'
             scope:               "email"
             options:
-                #display: popup
-                auth_type: rerequest
                 csrf: true
-
```

Comment:
So, we might need these social providers at some point, even if the configuration is not directly tied to the FOSUserBundle bundle. What do you think?

Reply:
Definitely. I’ve not really gotten to the configuration of the bundle. 

I had to remove it and reinstall it, so that the recipes from the *contrib* repository can be run.  That’s why the resource owners were removed.

But when we're working on the registration and login we'll definitely have to configure it properly.

Reply:
Definitely. I’ve not really gotten to the configuration of the bundle. 

I had to remove it and reinstall it, so that the recipes from the *contrib* repository can be run.  That’s why the resource owners were removed.

But when we're working on the registration and login we'll definitely have to configure it properly.